spec/controllers/redirects_controller_spec.rb

```diff
@@ -0,0 +1,63 @@
+describe RedirectsController do
+  describe 'auth' do
+    context 'when not logged in' do
+      it 'redirects user to login' do
+        get :new
+        expect(response).to_not be_success
```

Comment:
Would it not be better to say `expect(response).to redirect_to(login_path)` (or whatever the url helper is for the login path)?